2.5.0-rc.2 doesn't appear to compile the redirect middleware

```
# cat /etc/docker/registry/config.yml
version: 0.1
log:
  fields:
    service: registry
storage:
    cache:
        blobdescriptor: inmemory
    filesystem:
        rootdirectory: /var/lib/registry
http:
    addr: :5000
    headers:
        X-Content-Type-Options: [nosniff]
health:
  storagedriver:
    enabled: true
    interval: 10s
    threshold: 3
middleware:
  storage:
    - name: redirect
      options:
        baseurl: https://example.com/

# docker logs registry
time="2016-07-19T13:09:54Z" level=warning msg="No HTTP secret provided - generated random secret. This may cause problems with uploads if multiple registries are behind a load-balancer. To provide a shared secret, fill in http.secret in the configuration file or set the REGISTRY_HTTP_SECRET environment variable." go.version=go1.6.2 instance.id=34b77900-1852-4914-afdd-07b9db008fa7 version=v2.5.0-rc.2
time="2016-07-19T13:09:54Z" level=info msg="redis not configured" go.version=go1.6.2 instance.id=34b77900-1852-4914-afdd-07b9db008fa7 version=v2.5.0-rc.2
time="2016-07-19T13:09:54Z" level=info msg="using inmemory blob descriptor cache" go.version=go1.6.2 instance.id=34b77900-1852-4914-afdd-07b9db008fa7 version=v2.5.0-rc.2
time="2016-07-19T13:09:54Z" level=info msg="listening on [::]:5000" go.version=go1.6.2 instance.id=34b77900-1852-4914-afdd-07b9db008fa7 version=v2.5.0-rc.2
time="2016-07-19T13:09:54Z" level=info msg="Starting upload purge in 53m0s" go.version=go1.6.2 instance.id=34b77900-1852-4914-afdd-07b9db008fa7 version=v2.5.0-rc.2
panic: unable to configure storage middleware (redirect): no storage middleware registered with name: redirect
goroutine 1 [running]:
panic(0xd87000, 0xc8203be5d0)
        /usr/local/go/src/runtime/panic.go:481 +0x3e6
github.com/docker/distribution/registry/handlers.NewApp(0x7fd6837322c0, 0xc8203384b0, 0xc82006f8c0, 0x7fd6837322c0)
        /go/src/github.com/docker/distribution/registry/handlers/app.go:150 +0x100d
github.com/docker/distribution/registry.NewRegistry(0x7fd683732300, 0xc8203384b0, 0xc82006f8c0, 0xc82006f8c0, 0x0, 0x0)
        /go/src/github.com/docker/distribution/registry/registry.go:86 +0x2b0
github.com/docker/distribution/registry.glob.func1(0x14ca660, 0xc820328070, 0x1, 0x1)
        /go/src/github.com/docker/distribution/registry/registry.go:55 +0x2a9
github.com/docker/distribution/vendor/github.com/spf13/cobra.(*Command).execute(0x14ca660, 0xc820328020, 0x1, 0x1, 0x0, 0x0)
        /go/src/github.com/docker/distribution/vendor/github.com/spf13/cobra/command.go:495 +0x6d4
github.com/docker/distribution/vendor/github.com/spf13/cobra.(*Command).Execute(0x14ca800, 0x0, 0x0)
        /go/src/github.com/docker/distribution/vendor/github.com/spf13/cobra/command.go:560 +0x180
main.main()
        /go/src/github.com/docker/distribution/cmd/registry/main.go:23 +0x23
```

--- a/cmd/registry/main.go
+++ b/cmd/registry/main.go
@@ -13,6 +13,7 @@ import (
 	_ "github.com/docker/distribution/registry/storage/driver/gcs"
 	_ "github.com/docker/distribution/registry/storage/driver/inmemory"
 	_ "github.com/docker/distribution/registry/storage/driver/middleware/cloudfront"
+	_ "github.com/docker/distribution/registry/storage/driver/middleware/redirect"
 	_ "github.com/docker/distribution/registry/storage/driver/oss"
 	_ "github.com/docker/distribution/registry/storage/driver/s3-aws"
 	_ "github.com/docker/distribution/registry/storage/driver/s3-goamz"

